### PR TITLE
Feature/arts 930

### PR DESCRIPTION
**Pull request by darrengsmithson**
Adding parentName to site model to save calling the site service for every site in the response

## Description
Adding parentName to site model to save calling the site service for every site in the response.

### Changes
- [ARTS-930] - Adding parentName to site model to save calling the site service for every site in the response.

### Checklist:

- [ ] Branch name follows convention (feature/arts-###-short-name OR fix/arts-###-short-name)
- [ ] I have included a short-meaningful title, description and changes for this PR

[ARTS-930]: https://ndph-arts.atlassian.net/browse/ARTS-930

### CODE REVIEW

File: src/app/admin-sites-page/admin-sites-page.component.ts [hunk 1]
```diff
@@ -19,13 +19,6 @@ export class AdminSitesPageComponent implements OnInit {
     this.showModal = false;
     this.selectedSite = {};
     this.siteService.getSitesByRole('admin').pipe(first()).subscribe((sites) => {
-      sites.forEach(site => {
```
Comment (shajidissa) [inline src/app/admin-sites-page/admin-sites-page.component.ts:22]: @darrengsmithson why is this removed ?

**Review by shajidissa — CHANGES_REQUESTED** [review]

File: src/app/admin-sites-page/admin-sites-page.component.ts (hunk 1 above)
Comment (darrengsmithson) [reply src/app/admin-sites-page/admin-sites-page.component.ts:22]: we no longer need to call site service for each site to get the parent info

**Review by alexbaxter — APPROVED** [review]
Unfortunate that the wait time in the test had to be increased, but there you go.

**Review by shajidissa — APPROVED** [review]
